steps/codified-security-bitrise/1.0.0/step.yml

```diff
@@ -0,0 +1,53 @@
+title: Codified Security
+summary: This step uploads your app to Codified Security for automated mobile app
+  security testing
+description: This step uploads your app to Codified Security for automated mobile
+  app security testing
+website: https://codifiedsecurity.com
+published_at: 2017-04-19T19:05:03.975322686+02:00
+source:
+  git: https://bitbucket.org/codifiedldn/codified-security-bitrise.git
+  commit: b2d6cbb641d8f14e1f3632328cf56c8c857dc02a
+project_type_tags:
+- ios
+- android
+- xamarin
+type_tags:
+- android
```

Comment:
`type_tags` should be one, or more from this list:

- access control
- artifact info
- installer
- deploy
- utility
- dependency
- code sign
- build
- test
- notification

i would recommend to use `test`.